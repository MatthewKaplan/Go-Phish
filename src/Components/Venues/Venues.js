import React from "react";

export const Venues = ({ venue }) => {
  return (
    <div className="venues-component">
      <section className="venue-top">
        <h2 className="venue-name">{venue.name}</h2>
      </section>
      <section className="venue-bottom">
        <article className="venue-info">
          <h4>Location:</h4>
          <p>{venue.location}</p>
          <h4>Amount of shows at venue:</h4>
          <p>{venue.shows_count}</p>
          <br />
          <br />
          <section className="right-info">
            <h4>First time playing venue:</h4>
            <p>{venue.show_dates[0]}</p>
            <h4>Last time playing venue:</h4>
            <p>{venue.show_dates.slice(-1)[0]}</p>
          </section>
        </article>
      </section>
    </div>
  );
};
