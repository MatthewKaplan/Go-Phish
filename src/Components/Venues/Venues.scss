.venues-component {
  height: 350px;

  .venue-name {
    color: #eee;
    padding-top: 25px;
  }

  .venue-top {
    height: 100px;
    background-color: #772953;
    border-radius: 14px 14px 0 0;
  }

  .venue-bottom {
    position: static;
    text-align: left;
    height: 250px;
    background-image: url('https://i.imgur.com/9GB0xeq.jpg');
    background-repeat: no-repeat;
    background-size: cover;
    border-radius: 0 0 14px 14px;
  }

  .venue-info {
    position: relative;
    top: 20px;
    background-color: rgba(0, 0, 0, 0.706);
    border-radius: 12px;
    color: #fff;
    height: 210px;
    margin: auto;
    padding: 10px;
    width: 90%;
  }

  .right-info {
    text-align: right;
  }

  h4 {
    color: #C7142D;
    -webkit-text-stroke-width: .2px;
    -webkit-text-stroke-color: #004D95;
  }
}
