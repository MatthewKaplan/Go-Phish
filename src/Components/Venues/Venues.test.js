import React from "react";
import { Venues } from "./Venues";
import MockData from "../../Helpers/mockData";
import { shallow } from "enzyme";

const venueMock = MockData.mockVenue;

describe("Venues", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Venues venue={venueMock} />);
    expect(wrapper).toMatchSnapshot();
  });
});
