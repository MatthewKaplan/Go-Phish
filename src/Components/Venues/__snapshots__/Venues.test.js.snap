// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Venues should match snapshot 1`] = `ShallowWrapper {}`;
